modules/router-store/src/serializers/full_serializer.ts

```diff
@@ -26,6 +26,7 @@ export class FullRouterStateSerializer
       data: route.data,
       url: route.url,
       outlet: route.outlet,
+      title: route.title,
```

Comment:
In v15, `title` is required property

Reply:
🤔

Reply:
*`title` is required property in `ActivatedRouteSnapshot` class, not `Route` interface 😅

Reply:
Ahh ok. I was worried there for a minute 😄